src/Files.App/Actions/Git/GitPushAction.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Files.App.Commands;
+using Files.App.Contexts;
+
+namespace Files.App.Actions
+{
+	internal class GitPushAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext _context;
+
+		public string Label { get; } = "Push".GetLocalizedResource();
```

Comment:
I'd renamed the string to "GitPush".

Reply:
The string isn't specific to Git so I'd rather keep it generic in case we want to use it somewhere else.